boa/src/builtins/value/mod.rs

```diff
@@ -855,15 +849,15 @@ impl Display for ValueData {
             ValueData::Function(ref v) => match *v.borrow() {
                 Function::NativeFunc(_) => write!(f, "function() {{ [native code] }}"),
                 Function::RegularFunc(ref rf) => {
-                    write!(f, "function(")?;
+                    write!(f, "function{}(", if rf.args.is_empty() { "" } else { " " })?;
                     let last_index = rf.args.len() - 1;
                     for (index, arg) in rf.args.iter().enumerate() {
                         write!(f, "{}", arg)?;
                         if index != last_index {
                             write!(f, ", ")?;
                         }
                     }
```

Comment:
There is a bug here. When `rf.args.len() == 0`, then if you do `0usize - 1`, you get:
```
thread 'main' panicked at 'attempt to subtract with overflow', boa/src/builtins/value/mod.rs:859:38
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```
```suggestion
                    write!(f, "function{}(", if rf.args.is_empty() { "" } else { " " })?;
                    for (index, arg) in rf.args.iter().enumerate() {
                        write!(f, "{}", arg)?;
                        if index + 1 != rf.args.len() {
                            write!(f, ", ")?;
                        }
                    }
```